<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgViewer</class>
 <widget class="QWidget" name="ImgViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>10</y>
     <width>601</width>
     <height>400</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="nextBut">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevBut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
    </font>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </widget>
  <widget class="QLabel" name="rankText">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Image %d/%d: Ranked %.03f/10</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>601</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="indivRanks">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Individual ranks</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
